File: Aprendendo-Java/matrizes/src/soma_linhas.java
import java.util.Scanner;
import java.util.Locale;

public class soma_linhas {
    public static void main(String[] args){
        Locale.setDefault(Locale.US);
        Scanner sc = new Scanner(System.in);
        
        double[][] mat = new double[10][10];
        
        System.out.print("Qual a quatidade de linhas da matriz?: ");
        int m = sc.nextInt();
        double[] somaVet = new double[m];
        
        System.out.print("Qual a quatidade de columas da matriz?: ");
        int n = sc.nextInt();
        
        for(int i = 0; i < m;i++){
            System.out.println("Digite os elementos da "+(i+1)+"a linha");
            for(int j = 0;j < n;j++){                
             mat[i][j] = sc.nextDouble();
                
            }
         
        }
        for(int i = 0; i < m;i++){            
            for(int j = 0;j < n;j++){
             somaVet[i] += mat[i][j];
            
            }
        }   
        
        for(int i = 0; i < m;i++){
            System.out.println(somaVet[i]);
        }
    }    
}

File: Aprendendo-C/Exercício/soma_linhas.c
#include <stdio.h>
#include "gabsLib.h"

int main(){
    int i, j, n, m;
    
    printf("Qual a quantidade de linhas da matriz?: ");
    scanf("%d", &m);

    printf("Qual a quantidade de linhas da matriz?: ");
    scanf("%d", &n);

    double mat[m][n], somaVet[m];
    jumpln();

    for(i=0; i < m; i++){
        printf("Digite os elementos da %da linha \n",i+1);
        for (j = 0; j < n; j++){        
        scanf("%lf",&mat[i][j]);
        }        
    }

    for(int i = 0; i < m;i++){            
        for(int j = 0;j < n;j++){
             somaVet[i] += mat[i][j];            
            }    
        }

    printf("Vetor gerado: \n");
    for (i = 0; i < m; i++)
    {
        printf("%.1lf\n",somaVet[i]);
    }
         


    return 0;
}